docs/EN/Resources/clinician-guide-to-AndroidAPS.md

```diff
@@ -0,0 +1,101 @@
+
+#### Here are examples of the purple prediction lines, and how they might differ:
+
+![Purple prediction line examples](../images/Prediction_lines.jpg)
+
+#### Here are examples of different time frames that influence the needed adjustments to insulin delivery:
+
+#### Scenario 1 - Zero Temp for safety
+
+In this example, BG is rising in the near-term time frame; however, it is predicted to be low over a longer time frame. In fact, it is predicted to go below target *and* the safety threshold. For safety to prevent the low, AndroidAPS will issue a zero temp, until the eventualBG (in any time frame) is above threshold.
+
+![Dosing scenario 1](../images/Dosing_scenario_1.jpg)
+
+#### Scenario 2 - Zero temp for safety
+
+In this example, BG is predicted to go low in the near-term, but is predicted to eventually be above target. However, because the near-term low is actually below the safety threshold, AndroidAPS will issue a zero temp, until there is no longer any point of the prediction line that is below threshold.
+
+![Dosing scenario 2](../images/Dosing_scenario_2.jpg)
+
+#### Scenario 3 - More insulin needed
+
+In this example, a near-term prediction shows a dip below target. However, it is not predicted to be below the safety threshold. The eventual BG is above target. Therefore, AndroidAPS will restrain from adding any insulin that would contribute to a near-term low (by adding insulin that would make the prediction go below threshold). It will then assess adding insulin to bring the lowest level of the eventual predicted BG down to target, once it is safe to do so. *(Depending on settings and the amount and timing of insulin required, this insulin may be delivered via temp basals or SMB's).*
+
+![Dosing scenario 3](../images/Dosing_scenario_3.jpg)
+
+#### Scenario 4 - Low temping for safety
+
+In this example, AndroidAPS sees that BG is spiking well above target. However, due to the timing of insulin, there is already enough insulin in the body to bring BG into range eventually. In fact, BG is predicted to eventually be below target. Therefore, AndroidAPS will not provide extra insulin so it will not contribute to a longer-timeframe low. Although BG is high/rising, a low temporary basal rate is likely here.
+
+![Dosing scenario 4](../images/Dosing_scenario_4.jpg)
+
+### Optimizing settings and making changes 
+
+As a clinician who may not have experience with AndroidAPS or DIY closed loops, you may find it challenging to help your patient optimize their settings or make changes to improve their outcomes. We have multiple tools and [guides](http://openaps.readthedocs.io/en/latest/docs/Customize-Iterate/optimize-your-settings.html) in the community that help patients make small, tested adjustments to improve their settings. 
```

Comment:
if this page has been replicated in AAPS, let's point to it there.

Reply:
I haven't found a corresponding AndroidAPS page. Should it be replicated? I think that the OpenAPS page is good, and personally, I think that it's fine that not all pages are covered at both sites. Let me know if there is a desire to make the two documentations self-contained, in which case this should be replicated.